packages/@lwc/integration-karma/test/component/LightningElement.attachInternals/elementInternals/sanity/index.spec.js

```diff
@@ -0,0 +1,55 @@
+import { createElement } from 'lwc';
+import { ariaProperties, ariaAttributes } from 'test-utils';
+
+import ElementInternal from 'ei/component';
+
+if (process.env.NATIVE_SHADOW && typeof ElementInternals !== 'undefined') {
+    let elm;
+    beforeEach(() => {
+        elm = createElement('ei-component', { is: ElementInternal });
+        document.body.appendChild(elm);
+    });
+
+    afterEach(() => {
+        document.body.removeChild(elm);
+    });
+
+    describe('ElementInternals', () => {
+        it('should be associated to the correct element', () => {
+            // Ensure external and internal views of shadowRoot are the same
+            expect(elm.internals.shadowRoot).toBe(elm.template);
```

Comment:
TIL `elm.template` is exposed externally 🙃

Reply:
I [exposed](https://github.com/salesforce/lwc/blob/355cc8540bdb2f28103d799a5d83247da6c59cda/packages/%40lwc/integration-karma/test/component/LightningElement.attachInternals/elementInternals/sanity/ei/component/component.js) it just for testing 😃